File: PRAKTIKUM_3/PRAK301-2410817210004-Rahmad.py
a=int(input("masukan angka acak pertama : "))
b=int(input("masukan angka acak Ke-dua : "))
c=int(input("masukan angka acak ke-tiga : "))
if a<b & a<c:
    if b<c :
        print(f"{a}, {b}, {c}")
    else :
        print(f"{a}, {c}, {b}")
elif b<a & b<c:
    if a<c :
        print(f"{b}, {a}, {c}")
    else :
        print(f"{b}, {c}, {a}")
else :
    if b<a :
        print(f"{c}, {b}, {a}")
    else :
        print(f"{c}, {a}, {b}")

File: PRAKTIKUM_3/PRAK301-2410817210004-Rahmad.c
#include <stdio.h>
int main(){
int a,b,c;
printf("masukan angka acak pertama: ");
scanf("%d",&a);
printf("masukan angka acak ke-dua: ");
scanf("%d",&b);
printf("masukan angka acak ke-tiga: ");
scanf("%d",&c);
if (a<b && a<c){
   if (b<c){  
   printf("%d %d %d",a,b,c);
   }else{
   printf("%d %d %d",a,c,b);
   }
}else if(b<a && b<c){
   if (a<c){
      printf("%d %d %d",b,a,c);
   }else{
      printf("%d %d %d",b,c,a);
      
   }
}else{
   if(a<b){
      printf("%d %d %d",c,a,b);
   }else {
      printf("%d %d %d",c,b,a);
   }
}
}